starcli/__main__.py

```diff
@@ -24,7 +24,14 @@
 
 
 @click.command()
-@click.option("--lang", "-l", type=str, default="", help="Language filter eg: python")
+@click.option(
```

Comment:
Let's have this option (and spoken-language) able to be used multiple times, like `-l vim -l python`. Check out how it's done in `--topic`

Reply:
I have modified the ```--lang``` and ```--spoken-language``` and they can be used multiple times now.